src/dask_awkward/utils.py

```diff
@@ -38,6 +38,49 @@ def divisions_msg(name: str, *args: Array) -> str:
         return msg
 
 
+class TracerConversionError(TypeError):
+    """
+    This error occurs when a tracer is used in a context that requires a concrete
```

Comment:
"tracer" is a strange name for it. It's always a dask collection, usually a dak.Array, right?